qiskit_machine_learning/algorithms/distribution_learners/qgan/pytorch_discriminator.py

```diff
@@ -119,10 +119,9 @@ def get_label(self, x, detach=False):  # pylint: disable=arguments-differ
             torch.Tensor: Discriminator output, i.e. data label
         """
 
+        # TODO handle tensor case
```

Comment:
You have added TODOs where there were none before, I guess presuming pass was somehow a placeholder for doing something?

Reply:
I suppose yes but I thought it would be best to eliminate the pass until this case is implemented and handled. Added a TODO so that we do not forget.

Reply:
I think I am saying that it could well have been handled, that would have been my reading - i.e. it was explicit in there as not needing to do anything for that case - i.e. pass.